src/sentry/static/sentry/less/releases.less

```diff
@@ -85,3 +85,12 @@
   background-color: @green;
   margin-top: 9px;
 }
+
+/**
+* Connect Repository
+* ============================================================================
+*/
+
+.connect-repository {
```

Comment:
We actually don't need this bit as we have an `.align-center` utility class. I'm working on this section rn and will apply the fix.